docs/usage/types/number_types.md

```diff
@@ -15,123 +28,108 @@
 `decimal.Decimal`
 : Pydantic attempts to convert the value to a string, then passes the string to `Decimal(v)`
 
-`NegativeFloat`
-: allows a float which is negative; uses standard `float` parsing then checks the value is less than 0;
-  see [Constrained Types](#constrained-types)
+## Constrained Types
 
-`NegativeInt`
-: allows an int which is negative; uses standard `int` parsing then checks the value is less than 0;
-  see [Constrained Types](#constrained-types)
+Pydantic provides functions that can be used to constrain numbers:
 
-`PositiveFloat`
-: allows a float which is positive; uses standard `float` parsing then checks the value is greater than 0;
-  see [Constrained Types](#constrained-types)
+* `conint`: Add constraints to an `int` type.
+* `confloat`: Add constraints to a `float` type.
+* `condecimal`: Add constraints to a `decimal.Decimal` type.
 
-`PositiveInt`
-: allows an int which is positive; uses standard `int` parsing then checks the value is greater than 0;
-  see [Constrained Types](#constrained-types)
+Those functions accept the following arguments:
 
-`condecimal`
-: type method for constraining Decimals;
-  see [Constrained Types](#constrained-types)
+* `gt` (greater than)
+* `ge` (greater than or equal to)
+* `lt` (less than)
+* `le` (less than or equal to)
+* `multiple_of` (multiple of)
+* `strict` (whether to allow coercion from compatible types)
 
-`confloat`
-: type method for constraining floats;
-  see [Constrained Types](#constrained-types)
+There are other arguments for each function, which you can see in the API Reference of each function:
 
-`conint`
-: type method for constraining ints;
-  see [Constrained Types](#constrained-types)
+  * [`conint`](/api/types/#conint)
```

Comment:
Actually in the preview, this style of link does not appear to be rendering as a link. Suggest either using an in-place link [text](url) or moving the colon away from the brackets. Maybe replace with space-dash-space.

Reply:
Right.